### ISSUE TITLE
YellowBox warning when using a FlatList in a ScrollView

### ISSUE THREAD

**Issue by chetstone** [Purii/react-native-tableview-simple#194]
The example code always shows the TableView inside a ScrollView. (Except for the FlatList example on the README but this does not seem to be a complete example.) I wonder why this is necessary? When I omit the ScrollView in my code, only one Cell is rendered, the rest are not visible, even if the list is very short and would easily fit on the screen.

But if I do include ScrollView and a FlatList is included in the Table (or even outside the table but within the scrollview), I get a YellowBox warning:

_VirtualizedLists should never be nested inside play ScrollViews with the same orientation - use another VirtualizedList-backed container instead._

**Comment by chetstone** [comment]
Workaround: I was able to replace the top level ScrollView with a FlatList as in the [example here](https://github.com/GeekyAnts/NativeBase/issues/2947#issuecomment-617001078).
My code (it's a little more convoluted than just wrapping with ScrollView but it works):

```
  render: function() {
    const tableview = (
      <TableView>
        <Section
          style={styles.cellContainer}
        >
          <Cell
            key="206x"
            onPress={() => {
              this.onPress()
              });
            }}
            cellStyle="RightDetail"
            detail="Detail"
            accessory="DisclosureIndicator"
            title="Title"
          />
        </Section>
      </TableView>
    );

    return (
      <View style={{ flex: 1 }}>
        <FlatList
          style={{ width: '100%' }}
          nestedScrollEnabled
          data={['tableview', 'flatList']}
          keyExtractor={data => data}
          renderItem={({ item, index }) => {
            switch (index) {
              case 0:
                return tableview;
              case 1:
                return (
                  <MyFlatListComponent sessions={this.state.sessions} />
                );
              default:
                return null;
            }
          }}
        />
      </View>
    );
  }
```

**Comment by Purii** [comment]
Hi @chetstone!

It shouldn't be necessary to nest the component in a `ScrollView`.
Will release a fix soon.

**Comment by Purii** [comment]
Just released `v3.1.0` which should fix that issue.
Can you verify that @chetstone

**Comment by chetstone** [comment]
Yes that fixes it! Thanks for the quick response!
